scripts/lib/authority-and-program-updater.ts

```diff
@@ -386,20 +305,19 @@ export class AuthorityAndProgramUpdater {
 
   async updatePrograms() {
     // Create program content and (over)write file.
-    const filePath = PROGRAMS_DIR + `${this.state.toLowerCase()}_programs.ts`;
+    const filePath =
+      PROGRAMS_DIR + '/' + this.state.toUpperCase() + '/programs.json';
     const formatOptions = await prettier.resolveConfig(filePath);
     if (!formatOptions) {
       throw new Error(`Could not retrieve Prettier config for ${filePath}`);
     }
     formatOptions.filepath = filePath;
-    const tsFileContent = await createProgramsContent(
+    const JSONFileContent = await createProgramsContent(
```

Comment:
nittiest nit to have ever nitted: I tend to prefer lowercasing all acronyms that are at the beginning of a camelCase variable name as we do with [`nyJson`](https://github.com/rewiringamerica/api.rewiringamerica.org/blob/asa/convert-programs-to-json/scripts/ny-empower-income-limits.ts#L49-L50) and [`csvContent`](https://github.com/rewiringamerica/api.rewiringamerica.org/blob/asa/convert-programs-to-json/scripts/ny-empower-income-limits.ts#L49-L50)

Reply:
updated!